platform/linux-generic/odp_packet_io.c

```diff
@@ -455,6 +455,12 @@ int odp_pktio_config(odp_pktio_t hdl, const odp_pktio_config_t *config)
 
 	entry->s.config = *config;
 
+	entry->s.in_chksums.all_chksum = 0;
+	entry->s.in_chksums.chksum.ipv4 = config->pktin.bit.ipv4_chksum;
```

Comment:
Any plan to integrate 'drop' flags (drop_ipv4_err, drop_ipv6_err, drop_udp_err, drop_tcp_err) with the parser?
Will it make sense to use odp_pktin_config_opt_t (pktio_entry->s.config.pktin) instead of making a copy of csum flags in the new in_chksums field?